beacon-chain/powchain/service.go

```diff
@@ -146,6 +147,7 @@ type Service struct {
 	lastReceivedMerkleIndex int64 // Keeps track of the last received index to prevent log spam.
 	runError                error
 	preGenesisState         iface.BeaconState
+	bsUpdater 				BeaconnodeStatsUpdater
```

Comment:
Might need gofmt, not sure if indentation here is really off

Reply:
fixed